modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -68,7 +68,11 @@ object types {
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
     rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
-    oneTimeRewards: List[OneTimeReward] = List.empty
+    oneTimeRewards: List[OneTimeReward] = List(
+      // Transferring current balance of 43,430,294.88479231 from DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS
+      //  adjusted by 1e8 to build correct transaction amount
```

Comment:
And I don't think it got "adjusted", but that's what the value simply is. We don't have a precision per se in the codebase and I would say that we don't have it even on the network. The representation on the client apps just move the `,` so that it's easier to comprehend the amount.